scripts/core/datetime/datetime.ts

```diff
@@ -1,7 +1,6 @@
 import _ from 'lodash';
-import moment from 'moment';
 import {gettext} from 'core/utils';
-import momentTimezone from 'moment-timezone';
+import moment from 'moment-timezone';
```

Comment:
```
import moment from 'moment';
import momentTimezone from 'moment-timezone';
```

These are different packages, it would be good not to mix up the names.

Reply:
but 'moment-timezone' it's exporting everything 'moment' does, but adding timezone functions, it's meant to replace it

Reply:
Ok, I guess it's not a big deal then :)